modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/toolbarbuttons/plugin.js

```diff
@@ -13,12 +13,187 @@
  */
 
 (function () {
+	const HEADING_BOTH = 'Both';
+	const HEADING_COL = 'Column';
+	const HEADING_NONE = 'None';
+	const HEADING_ROW = 'Row';
+
 	const pluginName = 'toolbarbuttons';
 
 	if (CKEDITOR.plugins.get(pluginName)) {
 		return;
 	}
 
+	/**
+	 * Returns which heading style is set for the given table.
+	 *
+	 * @instance
+	 * @method getHeading
```

Comment:
We should remove `instance` and `method` annotations, these are not methods on an object. This goes for other functions in the file.